runtime/README.md

```diff
@@ -1402,14 +1523,15 @@ Command Code: `0x434D_4546` ("CMEF")
 | chksum                      | u32      |                                                          |
 | context size                | u32      | size of context                                          |
 | context                     | u8[...]  | This MUST come from the output of the `CM_ECDH_GENERATE` |
+| key usage                   | u32      | usage tag of the kind of key that will be output         |
```

Comment:
Perhaps worth noting here that the key usage must be compatible with a 384-bit key (like, we can't pass AES-256-GCM here).